src/objects.js

```diff
@@ -1,42 +1,57 @@
 const createPerson = (name, age) => {
-  // your code here
+return {
+  name: name,
+  age: age,
+}
 };
 
-const getName = object => {
-  // your code here
-};
+const getName = object => object.name
 
-const getProperty = (property, object) => {
-  // your code here
-};
 
-const hasProperty = (property, object) => {
-  // your code here
-};
+const getProperty = (property, object) => (object[property])
+  // return {
+  //   property.age,
+  //   property.name,
+
+
+const hasProperty = (property, object) =>
+  object.hasOwnProperty(property);
 
-const isOver65 = person => {
-  // your code here
-};
 
-const getAges = people => {
+const isOver65 = person => person.age > 65;
   // your code here
-};
 
-const findByName = (name, people) => {
+const getAges = people => people.map(person => person.age);
   // your code here
-};
 
-const findHondas = cars => {
+
+const findByName = (name, people) => people.find(element => element.name === name);
   // your code here
-};
+
+
+const findHondas = cars => cars.filter(object => object.manufacturer === 'Honda');
+  //I used find instead of filter which did not work. And called object 'element'!!?
```

Comment:
'element' is just a placeholder value to use inside the function, you can change it to whatever you want as its only used inside the function. Also, `find` will only return the first element it find that matches, whereas filter goes through the whole dataset

As an example you could re-write this function to be:
```
  const findHondas = cars => cars.filter(car => car.manufacturer === 'Honda');
```
Bit easier to read!